Makefile

```diff
@@ -37,7 +37,7 @@ install: build
 	cp $(OUTPUT) $(GOBIN)
 
 $(OUTPUT): $(GO_SOURCES) vendor VERSION
-	go build -o $(OUTPUT) -ldflags "$(LDFLAGS_VERSION)" main.go
+	go build -o $(OUTPUT) -ldflags "$(LDFLAGS_VERSION)" *.go
```

Comment:
Why is this necessary?

Reply:
To ensure that `main.go` and `configuration.go` are compiled together. When it specifies `main.go`, the code does not compile.

Reply:
we should explicitly specify both main.go and configuration.go in order to avoid future confusion

`make release` also needs to be updated.

Reply:
I'm not sure I understand *why* this is necessary, I thought `go build` found all required files (just the same way it knows how to find referenced packages).

Anyway, we could also change from "build these `.go` files" to "build this package"

Reply:
FWIW @ericbottard's suggestion of just building the package by not specifying any go source files at all in the `go build` invocations works fine for both the `make build` and `make release` targets.